packages/features/ee/workflows/lib/reminders/templates/customTemplate.ts

```diff
@@ -17,7 +18,12 @@ export type VariablesType = {
 };
 
 const customTemplate = async (text: string, variables: VariablesType, locale: string) => {
-  const timeWithTimeZone = `${variables.eventTime?.locale(locale).format("HH:mm")} (${variables.timeZone})`;
+  const translate = await getTranslation(locale ?? "en", "common");
+  const day = translate(dayjs(variables.eventDate).format("dddd").toLowerCase());
```

Comment:
I suggest using Intl instead of our own translations for weekdays/months 👍 - have a look at the booking page for examples of this